tests/integration/components/task-card-test.js

```diff
@@ -317,6 +317,6 @@ test('assignment dropdown does not render if user has no ability', function(asse
 
   assert.notOk(page.taskAssignment.select.triggerRenders, 'Dropdown trigger for assignment does not render.');
   page.taskAssignment.assignedUser.as((user) => {
-    assert.equal(page.taskAssignment.assignedUser.icon.url, 'test.png');
+    assert.equal(user.icon.url, 'test.png');
```

Comment:
Guessing this would be `page.something`. This is only passing because you're testing the data that you created yourself, not the component's data.